commandapi-annotations/pom.xml

```diff
@@ -54,18 +55,6 @@
 			<plugin>
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-compiler-plugin</artifactId>
-				<executions>
-					<execution>
-						<goals>
-							<goal>testCompile</goal>
-						</goals>
-						<configuration>
-							<annotationProcessors>
-								<annotationProcessor>dev.jorel.commandapi.annotations.Annotations</annotationProcessor>
-							</annotationProcessors>
-						</configuration>
-					</execution>
-				</executions>
```

Comment:
If this build plugin no longer has any `executions`, does it need to be declared anymore? Could this removal be expanded to also remove the whole `build` section?